docs/security-champion/4-learning-platform.md

```diff
@@ -0,0 +1,64 @@
+# Secure Code Learning platform
+
+We are testing out a secure coding learning platform. You as a champion are a perfect match, and that is why you get special merch by learning!
+
+## What can you expect?
+
+- Learning about security and secure coding in "hands on" sessions in a sandbox environment
+- View video lessons
+- Gain unique merch based off your belt level!
+- Contribute to the SCN
+- Have fun!
+- Bragging rights
+- Learn things that may help advance your career!
+
+!!! info
+    You can gain unique merch based on your belt level!
+
+There is a wide arrangement of subjects, and you can do them all if you wish! So there are tracks for:
+
+- AppSec
+- Docker security
```

Comment:
Could you add Azure to this list? (one of the advanced tracks was a bit rewarding)